viz.yaml

```diff
@@ -98,8 +98,8 @@ parameter:
   - 
     id: social-links
     github: https://github.com/USGS-VIZLAB/water-use-15
-    twitter: https://twitter.com/share?url=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=twitter_button_share
-    facebookLink: http://www.facebook.com/sharer.php?u=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=fb_button_share
+    twitter: https://twitter.com/share?url=https://owi.usgs.gov/vizlab/water-use-15/?utm_source=twitter_button_share%2F&text=The%20USGS%20studies%20water%20use%20in%20every%20county%20for%20the%20U.S.%20to%20manage%20current%20resources%20and%20prepare%20for%20future%20needs.%20Explore%20your%20county%20here&hashtags=water,usgs,wateruse
```

Comment:
`every county **in** the U.S.`?

Reply:
would it be `water use FOR every county IN the US`? in followed by in sounds weird

Reply:
Yeah, `water use for every county OF the US` maybe?

Reply:
I mean `water use in every county of the US`